README.md

```diff
@@ -93,195 +93,44 @@ Mapbox has a free tier, however there are paid options available. For most users
-
-3. [Ubuntu only as this is the default setup on macOS]: The postgresql authentication rules will need to change to allow easier access and the server to connect to postgres. This requires editing the `pg_hba.conf` file and restarting postgres. Here is more information about the configuration file and what changes are being made https://www.postgresql.org/docs/current/auth-pg-hba-conf.html.
-
-   Find the configuration file by running `sudo -u postgres psql -c "SHOW hba_file;"`. Edit the first uncommented line of that file to be
-      ```
-      local   all             postgres                                trust
-      ```
-   Then, add the following lines at the bottom:
-      ```
-      # Allow any user on the local system using local loopback TCP/IP connections
-      # to connect to any database with any database user name.
-      host    all             all             127.0.0.1/32            trust
-      # The same over IPv6.
-      host    all             all             ::1/128                 trust
-      ```
-   Finally, run `sudo service postgresql restart` for the changes to take effect.    ​
-
-4.  Enter psql client to check server success. If it works, then you can exit psql.
-
-    - macOS: `psql postgres`
-    - Ubuntu: `psql -U postgres`
-
-5.  Create and upgrade the local postgres database as well as populate the Data Catalog tables: `./scripts/db/postgres/dev/init_db.py --populate_indicators`
-
-    This command will create a database for each deployment defined in config, you can optionally run the command for just one deployment like `./scripts/db/postgres/dev/init_db.py <ZEN_ENV> --populate_indicators`. Also you can run just `./scripts/db/postgres/dev/init_db.py` to only upgrade the database and not populate Data Catalog.
-
-6. Create an admin user for your local web app like below.
-
-      ```
-      ./scripts/create_user.py -f <first name> -l <last name> -u <email> -p <password> -a -d postgresql://postgres:@localhost/{ZEN_ENV}-local
-      ```
+1. Prepare the database: `ZEN_ENV=<specify environment> docker compose run web /bin/bash -c "source venv/bin/activate && ./scripts/db/postgres/dev/init_db.py --populate_indicators"` e.g. : `ZEN_ENV=br docker compose run web /bin/bash -c "source venv/bin/activate && ./scripts/db/postgres/dev/init_db.py --populate_indicators"`
 
-### Hasura
+### Run Web Server
+1. Start your development environment: `DEFAULT_DRUID_HOST=http://<public production Druid server IP> ZEN_ENV=<specify environment> docker compose up` e.g. : `DEFAULT_DRUID_HOST=http://aws-druid.corp.clambda.com ZEN_ENV=br docker compose up`
+2. In a seperate terminal, create user account `docker compose exec web /bin/bash -c "source venv/bin/activate && ./scripts/create_user.py --u me@mydomain.com -p admin --first_name=admin --last_name=istrator -a"`
+3. Browse to website on [http://localhost:5000](http://localhost:5000) and log in with the credentials used in step 2.
```

Comment:
Hasura isn't loading correctly. When I run these steps and then navigate to AQT, there are the following console errors:
```
POST http://localhost:5000/api/graphql [HTTP/1.0 500 INTERNAL SERVER ERROR 387ms]

Warning: RelayResponseNormalizer: Payload did not contain a value for field `dimensionConnection: dimension_connection`. Check that you are parsing with the same query that was used to fetch the payload. [warning.js:30](webpack://zenysis/node_modules/fbjs/lib/warning.js)
Warning: RelayResponseNormalizer: Payload did not contain a value for field `fieldConnection: field_connection`. Check that you are parsing with the same query that was used to fetch the payload. [warning.js:30](webpack://zenysis/node_modules/fbjs/lib/warning.js)
Warning: RelayResponseNormalizer: Payload did not contain a value for field `categoryConnection: category_connection`. Check that you are parsing with the same query that was used to fetch the payload.
```

And when I go to http://localhost:8088/console/ or http://localhost:8080/console/ (the hasura console), there's an error

Reply:
Strangely I had to specify HASURA_GRAPHQL_ENABLE_CONSOLE: "true" to get the console. Very odd. We don't specify that in our main repo.

Other issue was because of hard coded hasura location. There's a **expletive deleted** piece of code: ```self.HASURA_HOST = getenv(
            'HASURA_HOST',
            global_config.HASURA_HOST if IS_PRODUCTION else 'http://localhost:8088',
        )``` in flask.py.

I've made note of it - prepare for a hotly debated PR in the near future where I rip out all of these defaults. As far as I'm concerned a missing configuration value is a reason for an exception, not a default.